integrations/upgrade/js-config.test.ts

```diff
@@ -230,24 +230,26 @@ test(
       "
       --- src/input.css ---
       @import 'tailwindcss';
-      @config '../tailwind.config.ts';
+
+      @theme {
+        --color-gray-50: oklch(0.985 0 none);
```

Comment:
I know this PR didn't introduce it, but I wonder if the "legacy" colors (v3) should still use the v3 values (hex) instead of `oklch(…)` 🤔